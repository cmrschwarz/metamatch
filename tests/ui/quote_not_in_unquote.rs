use metamatch::quote;

fn main() {
    let res = quote! {
        [<quote>]
        42
        [</quote>]
    };

    assert_eq!(res, 10);
}
