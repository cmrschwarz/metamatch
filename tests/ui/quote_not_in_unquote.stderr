error: redundant quote tag, arlready quoted
 --> tests/ui/quote_not_in_unquote.rs:5:11
  |
5 |         [<quote>]
  |           ^^^^^
